engine/lib/luajit-ffi-gen/src/impl_item/type_info.rs

```diff
@@ -1,6 +1,37 @@
-const RUST_TO_LUA_TYPE_MAP: [(&str, &str); 2] = [("IVec2", "Vec2i"), ("Vec3", "Vec3f")];
-const COPY_TYPES: [&str; 3] = ["IVec2", "WindowPos", "WindowMode"];
+const RUST_TO_LUA_TYPE_MAP: [(&str, &str); 5] = [
+    ("IVec2", "Vec2i"),
+    ("UVec2", "Vec2u"),
+    ("DVec2", "Vec2d"),
+    ("Vec2", "Vec2f"),
+    ("Vec3", "Vec3f"),
+];
 
+// TODO: find out different way to mark types as copyable
```

Comment:
Lets make sure this is tracked in an issue somewhere with all the other TODOs here (or just resolve this comment if that's already the case :))

Reply:
Added to tasklist #156